src/styles/bloks/_card.scss

```
@@ -0,0 +1,107 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: ($card-with + 48px) * 4;
+  box-sizing: content-box;
+
+  &__card {
+    width: $card-with;
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  background-color: #fff;
```

you created variables for colors, but not for all 🥺 
```suggestion
  background-color: $c-white;
```